pkgs/applications/audio/bucklespring/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+
+, legacy ? false
+, libinput
+, libudev
+
+, pkg-config
+, makeWrapper
+
+, openal
+, alure
+, libXtst
+, libX11
+}:
+
+with lib;
+
+stdenv.mkDerivation rec {
+  pname = "bucklespring";
+  version = "2021-01-21";
```

Comment:
```suggestion
  version = "unstable-2021-01-21";
```

Reply:
i think the date adequately implies it's not a tagged release
i also think `unstable` when used, should be part of pname, as in `kicad-unstable`

Reply:
Unstable should only be part of the name if there are two packages of the same software and no date does not imply it is untagged because it could be the upstream version number.

Reply:
`git-*` is significantly shorter and doesn't suggest instability
repology also seems to parse the date correctly and ignore it due to the `git-` part

Reply:
> git-* is significantly shorter and doesn't suggest instability

but we are not doing this and introducing multiple ignored prefixes just makes things harder.

Reply:
I'd agree with you in cases where there are no tagged releases, but there _are_ tagged releases, so this should follow the standard unstable- rule.

(I'm not saying the rule makes 100% sense and it's definitely worthwhile examining it, but a packaging PR isn't really the place to do that)

Reply:
yea, `version = git-*` seems to consistently be projects without tags